src/Container/ApplicationConfigInjectionDelegator.php

```diff
@@ -117,6 +117,7 @@ public static function injectPipelineFromConfig(Application $application, array
         }
 
         // Create a priority queue from the specifications
+        /** @var SplPriorityQueue $queue */
```

Comment:
createPriorityQueueReducer return type should be documented better here

Reply:
Yeah, it means using templates/generic types and I could get carried away at that

Reply:
Nah, just declare that it returns a `callable(...): SplPriorityQueue` 😜

This `@var` should go away

Reply:
so it did!